std/datetime.d

```diff
@@ -27488,6 +27503,13 @@ public:
                 // A bug on FreeBSD 9+ makes it so that this test fails.
                 // https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=168862
             }
+            else version(NetBSD)
+            {
+                //According http://mainfacts.com/world-timezones-time-zones/PPT :
+                //Actual zone of this location is PST now. It can be assigned to PPT too.
+                setTZEnvVar("America/Los_Angeles");
+                assert(LocalTime().dstName == "PPT" || LocalTime().dstName == "PDT");
```

Comment:
Have you actually seen "PPT" on a system somewhere? It is my understanding that PPT was used in 1945 and that's it: http://www.timeanddate.com/time/zone/usa/los-angeles

So, I see no reason to treat NetBSD differently from the other POSIX systems in this regard. They're all supposed to be using the same time zone files anyway. If you're actually seeing PPT, then I fully expect that the problem is that NetBSD has the same bug that FreeBSD does, in which case, I'd much prefer that rather than treat PPT as if it were correct, we'd skip this test on NetBSD for the moment and make sure that it's reported (which reminds me that I need to figure out how to push for the FreeBSD patch to be merged - it looks good to me, but it's been ignored thus far).